hail/python/hail/utils/struct.py

```diff
@@ -166,6 +166,23 @@ def drop(self, *args):
         -------
         :class:`.Struct`
             Struct without certain fields.
+
+        Examples
+        --------
+
+        Define a Struct `s`
+
+         >>> s = hl.Struct(food=8, fruit=5, bar=2, apple=10)
+
+        Drop one field from `s`
+
+         >>> s.drop(bar=2)
```

Comment:
I think this is not correct. What is the argument type `drop` requires? Remember drop is deleting a field. We don't need to assign a value at the same time.